.github/workflows/official-pr.yml

```diff
@@ -0,0 +1,63 @@
+name: Create official images PR
+
+on:
+  pull_request_target:
+    types:
+      - closed
+
+    paths:
+      - "**/Dockerfile"
+      - "**/architectures"
+      - "**/docker-entrypoint.sh"
+      - "generate-stackbrew-library.sh"
+      - "functions.sh"
+
+jobs:
+  pr:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'nodejs' && github.event.pull_request.merged_by != ''
+
+    steps:
+      - name: Checkout the docker-node repo
+        uses: actions/checkout@v2
+        with:
+          path: docker-node
+          fetch-depth: 50
+
+      - name: Checkout the official-images repo
+        uses: actions/checkout@v2
+        with:
+          path: official-images
+          repository: docker-library/official-images
+
+      - name: Generate Stackbrew for diff
+        run: |
+          cd docker-node
+          ./generate-stackbrew-library.sh > ../official-images/library/node
+
+      - name: Create PR in official-images
+        id: cpr
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.GITHUB_API_TOKEN }}
+          push-to-fork: nodejs-github-bot/official-images
+          path: official-images
+          branch: autopr
+          branch-suffix: short-commit-hash
+          commit-message: "Node: ${{ github.event.pull_request.title }}"
+          title: "Node: ${{ github.event.pull_request.title }}"
+          body: |
+            ${{ github.event.repository.full_name }}#${{ github.event.pull_request.number }}
+            @LaurentGoderre @pesho @PeterDaveHello @SimenB @Starefossen
+
+      - name: PR details
+        run: |
+          echo "Pull Request Number - ${{ steps.cpr.outputs.pull-request-number }}"
+          echo "Pull Request URL - ${{ steps.cpr.outputs.pull-request-url }}"
+
+      - name: Create PR comment
+        uses: peter-evans/create-or-update-comment@v1
+        with:
```

Comment:
Probably need `github-token: ${{secrets.GITHUB_TOKEN}}`

Reply:
Nevermind, I was thinking of another action, looks like this one already assumes this